Documentation/admin-guide/kernel-parameters.txt

```diff
@@ -5989,6 +5989,10 @@
 			non-zero "wait" parameter.  See weight_single
 			and weight_many.
 
+	sdw_mclk_divider=[SDW]
+			Specify the MCLK divider for Intel SoundWire buses in
+			case the BIOS does not provide the clock rate properly.
+
```

Comment:
this makes no sense to me. We have tons of kernel parameters that are for developers only. We don't want to maintain or advertise them as a supported configuration interface.